README.md

```diff
@@ -1,7 +1,46 @@
 ## Directory Structure
 
-*TO BE DECIDED*
+MVC Strucutre is being followed.
+
+`main.go`: Driver Code for all of backend.
+
+`models`: Database Models
+
+`controllers`: Various handlers for HTTP Requests
+
+`routes`: Definitions of Sub-routers
+
+`utils`: Middlewares and other files common to multiple functions
+
+`scripts`: Extra scripts (unrelated to the usual flow of the webapp)
+
+`docs`: Documentation
+
+## File Naming Convention
+
+- Separate file for each Model.
+
+- Controller functions to be grouped together by their routes.
+
+- Each Subroute has a separate file.
+
+In short, don't keep any surprises. Use groupings as per your discretion.
+
+## Dependencies
+
+```
+gorilla/mux
+jinzhu/gorm
+dgrijalva/jwt-go
+```
+
+Also uses `golanglint-ci` for linting code.
 
 ## Set up
```

Comment:
Can we have a shell file for building the files and testing the code, in case we plan to have tests? Shell scripts would make the job much easier and concise

Reply:
Should I write a Makefile?
Saw many other go projects doing the same.
I think using 2 commands: `make` and `make test` would be enough for us.

Reply:
Yeah makefile sounds great!